sdks/python/apache_beam/testing/load_tests/load_test_metrics_utils.py

```diff
@@ -138,8 +143,25 @@ def as_dict(self):
 class CounterMetric(Metric):
   def __init__(self, counter_dict, submit_timestamp, metric_id):
     super(CounterMetric, self).__init__(submit_timestamp, metric_id)
-    self.value = counter_dict.committed
     self.label = str(counter_dict.key.metric.name)
+    self.value = counter_dict.committed
+
+
+class DistributionMetrics(Metric):
+  def __init__(self, dists, submit_timestamp, metric_id, key):
+    super(DistributionMetrics, self).__init__(submit_timestamp, metric_id)
+    self.dist_label = str(key)
+    self.values_list = self._get_list(dists)
+
+  def _get_list(self, dists):
```

Comment:
Consider a more mnemonic name for this helper - a reader of the code would have to look into implementation to understand what this method does.

Reply:
Ok, actually I removed it.